### DISCUSSION TITLE
Support for direct use of tailwind classes in mdx

### DISCUSSION THREAD

**Discussion by Jeff-Russ** [timlrx/tailwind-nextjs-starter-blog#404] [Ideas]
I noticed that, if Tailwind classes are added to HTML(JSX) elements within `.mdx` files, some of these classes don't render because they aren't defined in the CSS delivered by the app. Other CSS classes *do* get defined but I think that's because they're referenced elsewhere in React components within `.js`/`.jsx` files that are analyzed by `postcss` to be needed for the particular url path. That was what I suspected at least. It doesn't seem that use of Tailwind classes within the `.mdx` files is a stated feature but  they *do*  appear in [data/blog/pictures-of-canada.mdx](https://raw.githubusercontent.com/timlrx/tailwind-nextjs-starter-blog/master/data/blog/pictures-of-canada.mdx). 

This was functionality I needed. For this to work, I added to [tailwind.config.js](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/tailwind.config.js), adding `'./data/**/*.mdx'` to its `module.exports.content` array, which appeared to do the trick. 

```jsx
content: [ './pages/**/*.js', './components/**/*.js', './layouts/**/*.js', './lib/**/*.js', './data/**/*.mdx'],
```

Just thought I'd mention what I found in case the developers want this functionality! Maybe there is a better way, but this seems to work for me for now since I'm not using any remotely hosted `.mdx`.

**Comment by timlrx** [comment]
Thanks, good idea. I will add it to tailwind.config for the template.
